HW3/DZ4.py

```diff
@@ -0,0 +1,8 @@
+numder = input("Введіть 4 числа: ")
+Product = int(numder[0]) * int(numder[1]) * int(numder[2]) * int(numder[3])
+print("Добуток числа:  ", Product)
+print(f"Число наоборот:  {numder[3]}{numder[2]}{numder[1]}{numder[0]}")
+print("Відсуртовано:  ", numder[0], numder[1], numder[2], numder[3])
+reverse = list(numder)
+reverse = sorted(reverse)
+
```

Comment:
number = input("Enter a four digit number:")

print(f'The product of numbers is: {int(number[0]) * int(number[1]) * int(number[2]) * int(number[3])}')

print(f'Reversed number: {number[::-1]}')

print(f'Sorted list: {list("".join(sorted(number)))}\n')